app/assets/javascripts/events.js.coffee

```diff
@@ -1,36 +1,41 @@
 jQuery ->
   $("#event_title").focus()
-  $table = $('#events').dataTable
+  $eventTable = $('#events').dataTable
+    'dom': "fr<'event-table-info't>ip",
```

Comment:
fr<'event-table-info't>ip
What is the `t` just inside the `>ip' ?

Reply:
it stands for table. All of those letters stand for a particular section of dataTables() and how it renders the table itself. It means it will add a div called 'event-table-info' inside of the table when compiling the html.

Reply:
<div class='event-table-info'>
</div>

Reply:
I expected the `t` to be outside the `>`, but I could easily be wrong.
I was looking at this page
https://datatables.net/reference/option/dom

Reply:
If you put the t on the outside it will not render the div in the proper location. I can get away with having to do this by using a really long selector to append to but I was trying to avoid that.

Reply:
Sorry, I should have been clearer. I was not suggesting another approach, just trying to learn !